inlang/source-code/website/package.json

```diff
@@ -101,5 +101,10 @@
 	"peerDependencies": {
 		"express": "^4.18.2"
 	},
-	"license": "Apache-2.0"
+	"license": "Apache-2.0",
+	"description": "This module is a monolith that contains inlang's website and editor to manage translations.",
+	"version": "1.0.0",
+	"main": "index.js",
+	"keywords": [],
+	"author": ""
```

Comment:
1. Why did you add those properties? 
2. Remove them. This pacakge is not published to NPM. Main doesn;'t even exist.